editor/src/components/editor/store/store-deep-equality-instances-3.spec.ts

```diff
@@ -265,6 +265,7 @@ describe('SpecialSizeMeasurementsKeepDeepEquality', () => {
     gap: 11,
     flexDirection: 'column',
     justifyContent: 'center',
+    alignContent: 'auto',
```

Comment:
Looks like you've updated the spec but not the implementation?

Reply:
yeah, I wonder why TS didn't warn about it. fixed!

Reply:
It definitely did on the CI server! :)